source/process/release-process.md

```diff
@@ -135,41 +170,43 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Tags a new release (e.g. 1.1.0) and runs an official build which should be essentially identical to the last RC. 
 3. PM:
     - Post in Release Discussion with links to the EE and Team Edition bits.
-    - Submit GitLab ticket to take next Mattermost version in the Omnibus using [template](https://gitlab.com/gitlab-org/omnibus-gitlab/issues/1241)
-    - Update the mattermost.org/download page
-    - Add the download links to http://docs.mattermost.com/administration/upgrade.html#version-archive
+    - Update the [Mattermost server download page](mattermost.org/download)
+    - Add the download links to [upgrdae guide](http://docs.mattermost.com/administration/upgrade.html#version-archive)
     - Contact owners of [community installers](http://www.mattermost.org/installation/) or submit [PRs](https://github.com/tommyvn/mattermost-heroku/commit/94f7c5c0c5d7d2672fb6d62b6a560b4b5c1b5131) to update install version number.
     - Close GitHub meta ticket for the release.
     - Merge the docs release branch to master and verify all changes on docs.mattermost.com once the build is up. Submit a correction PR for any incorrect formatting or other errors missed during the intitial review.
     - Update [MVP page](https://www.mattermost.org/mvp/) with the most valuable contributor of the release.
-    - Submit [GitLab MR](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/871/diffs) to take next Mattermost version in the Omnibus.
+    - Submit GitLab MR for `config/software/mattermost.rb` to take next Mattermost version in the Omnibus. The MR should include changes to the `default_version` and `md5` sum (see [example](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/933/diffs)
     - Update [Docker preview image to latest version](https://github.com/mattermost/mattermost-docker-preview/commit/1c34195d70b26bd4c82b5ef4fa0ebaf421096881).
 4. Marketing:
     - Finalize Mailchimp email blast
     - Finalize blog post for mattermost.org and set timer for 06:00 PDT on the day of release.
     - Queue Tweet announcement
-    - Find [GitLab release announcement merge request](https://gitlab.com/gitlab-com/www-gitlab-com/merge_requests) and post proposal for [GitLab Mattermost update text](https://gitlab.com/gitlab-com/www-gitlab-com/merge_requests/1633/diffs#efdba966a222d7071509e0dd4f39de5c9d7c1200_0_185)
```

Comment:
This isn't a redundant step, 
1. There's an MR to add the latest version of Mattermost to the latest GitLab release (above)
2. There needs to be a COMMENT by Mattermost on the MR for the GitLab to include a mention of Mattermost in the GitLab BLOG POST for their next release.